com.endlessm.DiscoveryFeed.json.in

```diff
@@ -20,6 +20,7 @@
         "--filesystem=host:ro",
         "--own-name=com.endlessm.DiscoveryFeed",
         "--talk-name=ca.desrt.dconf",
+        "--talk-name=com.endlessm.EknServices3.SearchProviderV3",
```

Comment:
Does not look like it belongs to this PR, but we should have this!

Reply:
Indeed, this is something I spotted whilst I was doing some testing here. Probably easier to just leave it here.